index.js

```diff
@@ -20,35 +20,50 @@ const exportDefaultFrom = require('@babel/plugin-proposal-export-default-from').
 const proposalOptionalChaining = require('@babel/plugin-proposal-optional-chaining').default;
 const removePropTypes = require('babel-plugin-transform-react-remove-prop-types').default;
 
-const { browserlist, legacyBrowserList } = require('./browserlist');
-
-module.exports = () => ({
-  env: {
-    modern: {
-      presets: [
-        [envPreset, {
-          targets: {
-            browsers: browserlist,
-            node: '12.16.1',
-          },
-        }],
-      ],
+const { browserList, legacyBrowserList } = require('./browserlist');
+
+module.exports = (api = {}, opts = {}) => {
+  const serverOnly = opts.serverOnly || (api.env && api.env('server'));
+  const isModern = opts.modern || (api.env && api.env('modern'));
+
+  const targets = {
+    node: '12.16.1',
```

Comment:
Should we also bump this to `12.16.3` now?

Reply:
maybe `current` would be better, then we don't have to keep updating